sgkit/tests/test_popgen.py

```diff
@@ -516,6 +516,7 @@ def test_Garud_h__raise_on_no_windows():
         Garud_H(ds)
 
 
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
What is this suppressing?

Reply:
This is suppressing a warning caused by dividing by zero (i.e., zero observations for a cohort because all observations are `nan`) which results in that `nan`s seen in the expected results. The previous implementation worked around this edge case to avoid the warning, however,  allowing the warning seems to be more consistent with what we do elsewhere. I'm happy reintroduce the workaround if it's preferred.

Reply:
I think issuing the warning is the preferred way to do things. Thanks for the explanation!